config/manifests/bases/kubernetes-image-puller-operator.clusterserviceversion.yaml

```diff
@@ -53,7 +67,7 @@ spec:
     name: Ilya Buziuk
   - email: tgeorge@redhat.com
```

Comment:
@tolusha could you please add yourself to the maintainer list and drop Tom George?

Reply:
Sure